dev/watch.js

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env node
+
+const path = require('path');
+
+const chalk = require('chalk');
+const browserSync = require('browser-sync').create();
+const bsConfig = require('./bs-config');
+
+// ********************************** JAVASCRIPT **********************************
+
+// webpack watch task always performs an intial bundle, and we don't want browsersync
+// listening at the point. so we use this flag to know whether any change webpack reports
+// is the initial bundle, or subsequent file changes
+let INITIAL_BUNDLE = true;
+
+// just a bit of visual feedback while webpack creates its initial bundles.
+// fakes a listr step
+const ora = require('ora');
+
+const wpNotification = ora({
+    text: 'Create initial webpack bundles',
+    color: 'yellow',
+});
+wpNotification.start();
+
+const webpack = require('webpack');
+const webpackBundler = webpack(require('../webpack.config.js')());
+
+webpackBundler.watch({
+    ignored: /node_modules/,
+}, (err, stats) => {
+    if (err) {
+        // log any unexpected error
+        console.err('webpack error:', err);
```

Comment:
Would be cool to implement [a nice logging module](https://github.com/jonnyreeves/js-logger) one day!